evmcore/tx_validation_test.go

```diff
@@ -245,117 +128,156 @@ func TestValidateTx_GasPriceAndTip_RejectsTxWith(t *testing.T) {
 	}
 }
 
-func TestValidateTx_Gas_RejectsTxWith(t *testing.T) {
-	for name, tx := range getTxsOfAllTypes() {
-		t.Run(fmt.Sprintf("current max gas lower than tx gas/%v", name), func(t *testing.T) {
-			opt := getTestTransactionsOption()
-			opt.currentMaxGas = 1
+func TestValidateTxStatic_AuthorizationList_RejectsTxWith(t *testing.T) {
```

Comment:
nit
```suggestion
func TestValidateTxStatic_AuthorizationList_RejectsTxWithEmptyAuthorization(t *testing.T) {
```

remove superfluous t..Run